PWGGA/GammaConv/AliAnalysisTRDEfficiency.h

```diff
@@ -6,118 +6,52 @@
+        AliTRDonlineTrackMatching* fOnline;      // For rating trdtrack to esdtrack matching
+        //AliAnalysisTaskESDfilter*  esdfilter;
+        AliConversionSelection* fConvsel;        // to get 'make' pi0
 
-        //tracks
-        TH1F*                   fhtxv;           // track->Xv()
-        TH1F*                   fhtyv;
-        TH1F*                   fhtzv;
-        
-        // gamma
-        TH1F*                   fhgpt;
-        TH1F*                   fhgpttrd;
-        TH2F*                   fhgRpt;
-        TH2F*                   fhgRpttrd;      // actually hqu but too late to change
-        
-        TH2F*                   fhgMinvM;       // comparing the GetPhotonMass and the GetInvMass functions
-        TH1F*                   fhgR;
-        TH2F*                   fhgptM;
-        TH2F*                   fhgptMhqu;
-        
-        TH2F*                   fhgptQ;     // pt vs photon Quality
-        TH2F*                   fhgptQhqu;
-        TH2F*                   fhgetaphi;
-        TH2F*                   fhgetaphihqu;
-        
-        TH2F*                   fhgxy;
-        TH2F*                   fhgxyhqu;
-        
-        // v0 duagther particles
-        TH1F*                   fhdn;
-        TH1F*                   fhdpt;
-        
-        // n dimensional
-        THnSparse*              fhna;
-        THnSparse*              fhnp;
-        THnSparse*              fhnhqu;
-        
-        // all of these events have a reconstructed photon
-        THnSparse*              fhgevent2;      // events with 
-        THnSparse*              fhgevent3;
-        THnSparse*              fhgevent4;
-        THnSparse*              fhgevent5;
-        THnSparse*              fhgevent6;
-        THnSparse*              fhgevent7;
-        THnSparse*              fhgevent8;
-        THnSparse*              fhgevent9;
-        
-        //event counter
-        TH1F*                   fhgevent;    // event counter
-        TH1F*                   fhevent;
-        
-        //track the events
-        THnSparse*              fhtrckvnt;   // track event
-        THnSparse*              fhtrckvnthqu;// track hqu event
-        TH1*                    fhtrvnt;
-        TH1*                    fhtrvnthqu;
-        TList*                  lsttrckvnt;
-        TList*                  lsttrckvnthqu;
-        //AliConversionPhotonCuts fConversionCuts;
-        
-        TH2F*                   fhgetaphi1;
-        TH1F*                   fhgR1;
-        TH1F*                   fhgpt1;
-        TH2F*                   fhgetaphi5;
-        TH2F*                   fhgetaphi8;
-        TH2F*                   fhgetaphi9;
-        
         AliAnalysisTRDEfficiency(const AliAnalysisTRDEfficiency&); // not implemented
         AliAnalysisTRDEfficiency& operator=(const AliAnalysisTRDEfficiency&); // not implemented
 
-        ClassDef(AliAnalysisTRDEfficiency, 1);
+        ClassDef(AliAnalysisTRDEfficiency, 4);
```

Comment:
if there is a new PR, no matter how many commits you have done locally, it is enough to increment by 1